third_party/aws-sdk-cpp/aws-sdk-cpp.bzl

```diff
@@ -114,78 +124,14 @@ cmake(
 )
 
 cc_library(
-    name = "aws-sdk-cpp_ubuntu",
+    name = "aws-sdk-cpp",
     deps = [
-        ":aws-sdk-cpp_cmake_ubuntu",
```

Comment:
That's a big block removed. Are you sure we can do that safely? 
All that building options and libraries...

Reply:
I see it's also in former ubuntu target. If this is all the same and we just remove duplicates then I'm okay with it.

Reply:
We remove duplicates.